File: strings/name_decorator.c
//Lila Shearer, Name Decorator C
#include <stdio.h>
#include <string.h>
char name[50];
char option[50] = "<><>";
char option_part_two[50] = "<><>";


int main(void){
   printf("Welcome to name decorator!\nThis program is deisgned to take your name and add fun decorations to it.\nWhat is your first name?\n");
   scanf("%s", name);
   strcat(option,name);
   strcat(option,option_part_two);
   printf("%s\n", option);
   
    return 0;
}

File: strings/name_decorator.py
# Lila Shearer, Name Decorator Python

decoration_one = "<><>"
decoration_two = "<><>"
messageone = "\nAlright, "
messagetwo = ". Here is your decorated name:\n"
print("\nHello! Welcome to Name Decorator.\nThe purpose of this program is to add decorations to your name.\n")

name = input("What is your first name?\n").strip().capitalize()

print(messageone+name+messagetwo)

print(decoration_one+name+decoration_two)
print("\n")